packages/ui/src/components/NavBar/NavBar.tsx

```diff
@@ -1,55 +1,26 @@
-import { BlurView } from 'expo-blur';
-import React, { PropsWithChildren } from 'react';
-import { Platform } from 'react-native';
+import React from 'react';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
-import { useTheme } from '../../core';
 import { XStack } from '../../core';
 import { View } from '../View';
 
-export const navHeight = 50;
-
 const NavBar = React.memo(function NavBar(props: {
   children: React.ReactNode | React.ReactNode[] | null | undefined;
 }) {
   const { bottom } = useSafeAreaInsets();
 
   return (
     <View
-      position="absolute"
-      width={'100%'}
-      bottom={0}
-      paddingTop={'$m'}
-      height={navHeight + bottom}
-      backgroundColor={Platform.OS === 'ios' ? 'transparent' : '$background'}
+      width="100%"
+      backgroundColor="$background"
+      paddingTop="$s"
+      paddingBottom={bottom}
```

Comment:
The bottom of the navbar is cut off on android (same screenshot from earlier)
![Screenshot_1717682814](https://github.com/tloncorp/tlon-apps/assets/1221094/7fa76672-af60-408f-8450-af2ac3a90640)

Reply:
Padding up conditionally for Android.